olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -430,33 +430,33 @@ spec:
                       - name: SERVICE_MONITOR_RBAC_MANIFEST_PATH
                         value: "/var/run/ko/monitoring/rbac-proxy.yaml"
                       - name: "IMAGE_queue-proxy"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.2.0:knative-serving-queue"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.3.2:knative-serving-queue"
```

Comment:
Images are not built yet.

Reply:
@skonto they are versioned as `1.3.0` - see: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v1.3__410.yaml#L223

Reply:
Ah, you forced pushed the commit and updated to 1.3.0 

yes, those are available and I can `docker pull` 'em